build/builder.go

```diff
@@ -54,6 +55,15 @@ type Branch struct {
 	Version string
 }
 
+type CTagsParserType = uint8
+
+const (
+	UnknownCTags CTagsParserType = iota
+	NoCTags
+	UniversalCTags
+	ScipCTags
+)
```

Comment:
can we give these an ascii char value to make it easier to understand when reading things like the languagemap argument. eg `? ! u s` respectively. I know that seems arb, but less arb than the iota values. Maybe `? !` are a bad idea since it makes it harder to copy paste into shells, but something along these lines feels good.

Edit: I guess unknown can just be null (0) since we shouldn't be able to specify it, but its what we get if we lookup in the languagemap and its missing.

Reply:
That's a great point, I do this though to keep it matching with the PR upstream at https://github.com/sourcegraph/sourcegraph/pull/50568 - maybe I can increment these by 97 or something in the languagemap argument to make these visible ASCII chars and thus more debuggable/pastable.